src/main/java/com/googlecode/gmail4j/javamail/JavaMailGmailMessage.java

```
@@ -331,4 +340,96 @@ public MessageHeaderInfo getMessageHeaderInfo() {
         }
         return headerInfo;
     }
+    
+    @Override
+    public void addAttachement(File file) {
+    	try {
+    		/* Check if email has already some contents. */
+    		Object content;
+    		try {
+    			content = this.source.getContent();
+			} catch (IOException e) {
```

Use `log.warn` to state what's happening. Swallowing an exception is a bad practice.